numpy/core/code_generators/cversions.txt

```diff
@@ -73,4 +73,6 @@
 0x00000011 = ca1aebdad799358149567d9d93cbca09
 
 # Version 18 (NumPy 2.0.0)
-0x00000012 = 5af92e858ce8e95409ae1fcc8f508ddd
+# Many API deprecations have been finalized
+# Add datetime conversion functions GH#21199
```

Comment:
If we're not enumerating every single change for numpy 2.0 I'm happy to remove these.

Reply:
Yah, lets not, already a few removals too.